nvflare/app_opt/confidential_computing/gpu_authorizer.py

```diff
@@ -61,8 +63,9 @@ def __init__(self, verifier_url="https://nras.attestation.nvidia.com/v1/attest/g
         self._can_generate = True
         self.client = attestation.Attestation()
         self.client.set_name("nvflare_node")
-        nonce = uuid.uuid4().hex + uuid.uuid1().hex
-        self.client.set_nonce(nonce)
+        self.my_nonce_history = NonceHistory(100)
+        self.seen_nonce_history = NonceHistory(5000)
```

Comment:
could we either introduce constants for these default numbers or make them part of the init args of this class?

Reply:
Both will be specified by one argument in the __init__ now.